gno.land/pkg/integration/testing_integration.go

```diff
@@ -107,12 +117,25 @@ func setupGnolandTestScript(t *testing.T, txtarDir string) testscript.Params {
 				env.Values["_logger"] = logger
 			}
 
-			// Setup "test1" default account
+			// test1 must be created outside of the loop below because it is already included in genesis so
+			// attempting to recreate results in it getting overwritten and breaking existing tests that
+			// rely on its address being static.
```

Comment:
Hot take: `test1` shouldn't be premined upon node starts.

It's completely unexpected behavior, the chain should be fresh

Reply:
Almost all txtar tests rely on `test1` existing, so making this change wold require all tests be updated with `adduser`.  Since this PR is concerned with the `adduser` command, I think that removing `test1` as an account created by default is out of scope here.